package.json

```diff
@@ -56,6 +56,8 @@
     "test": "npm run lint && npm run lint-css",
     "ballotTest-Android": "./node_modules/.bin/wdio ./tests/browserstack/ballotAndroid.wdio.config.js",
     "ballotTest-Browser": "./node_modules/.bin/wdio ./tests/browserstack/ballotBrowser.wdio.config.js",
+    "valuesTest-Android": "./node_modules/.bin/wdio ./tests/browserstack/valuesAndroid.wdio.config.js",
```

Comment:
Could you please add these scripts to the "To run the tests, run one or more of the following" section of: https://github.com/wevote/WebApp/blob/develop/docs/testing/README_TESTING.md